src/Rings/mpoly-ideals.jl

```diff
@@ -2072,8 +2085,36 @@ function flag_pluecker_ideal(F::Field, dimensions::Vector{Int}, n::Int)
   genz = [sum(t) for t in T]
   filter!(!iszero, genz)
   gg = unique(x ->  divexact(x, canonical_unit(x)), genz)
+  I  = ideal(gg)
 
-  return ideal(gg)
+  !minimal && return I 
+  # Computes the groebner_basis of the ideal
+  converted_generators = collect(groebner_basis(I; ordering = degrevlex(base_ring(I))))
+  o = degrevlex(R)
```

Comment:
seems like you haven't defined R in this scope, but maybe you should define the ordering `o` before you defined converted_generators

Reply:
It's defined on line 2073 I believe.